packages/cyberstorm/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from "react";
+
+import { faHouse, faSlashForward } from "@fortawesome/pro-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { LinkingProvider, LinkLibrary } from "./LinkingProvider";
+
+import styles from "./componentStyles/Breadcrumb.module.css";
+import { IndexLink } from "./Links";
+
+interface Crumb {
+  label: string | typeof FontAwesomeIcon;
+  LinkComponent?: (props: any) => React.ReactElement | null; // eslint-disable-line @typescript-eslint/no-explicit-any
+  LinkProps?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+interface Props {
+  linkLibrary: LinkLibrary;
+  parts: Crumb[];
+}
+
+/**
+ * Navigational bread crumbs defining where the user is at the site.
+ */
+export const BreadCrumb: React.FC<Props> = (props) => {
+  if (!props.parts) {
+    return null;
+  }
+  const parts = [...props.parts];
+  const leaf = parts.pop();
+
+  return (
+    <div className={styles.root}>
+      {parts.map((crumb, i) => (
+        <Fragment key={`${crumb.label}-${i}`}>
+          <MidCrumb linkLibrary={props.linkLibrary} crumb={crumb} />
+          <Separator />
+        </Fragment>
+      ))}
+
+      <LeafCrumb crumb={leaf} />
+    </div>
+  );
+};
+
+export const DefaultHomeCrumb: Crumb = {
+  LinkComponent: IndexLink,
+  label: (
+    <FontAwesomeIcon fixedWidth icon={faHouse} className={styles.separator} />
+  ),
+};
+
+const MidCrumb: React.FC<{ linkLibrary: LinkLibrary; crumb: Crumb }> = (
+  props
+) => {
+  const { linkLibrary } = props;
```

Comment:
Why is this and the next line two separate lines?

Reply:
Probably didn't realize I could do it in the same line. Removed the second one, since it was no longer needed.